mappers/tourist-attraction.js

```diff
@@ -32,7 +32,7 @@ export default function mapTouristAttractions(records, translations) {
       if (type) {
         publicG.add(sym(attractionUri), RDF('type'), sym(type));
       } else {
-        console.error(`Cannot map information group value '${record['information_group']}' for record ${recordId}`);
+        errorLogger('information group', record['information_group'], recordId);
```

Comment:
Pass the same field name as used to lookup the value as first argument. I know it was different before, but old code is wrong.